src/main/java/org/junit/runners/ParentRunner.java

```diff
@@ -206,15 +215,33 @@ private boolean areAllChildrenIgnored() {
     }
 
     /**
-     * Returns a {@link Statement}: run all non-overridden {@code @BeforeClass} methods on this class
+     * Returns a {@link Statement}: run all non-overridden {@code @Ignore} methods on this class
+     * and superclasses before executing {@code next}.
+     */
+    protected Statement withIgnores(final RunNotifier notifier) {
+        return new Statement() {
+            @Override
+            public void evaluate() throws Throwable {
+                for (final T each : getFilteredChildren()) {
```

Comment:
It is against our coding style. We generally only make locals final if they need to be in order to be accessed in an anonymous class. The extra "noise" of making the field final isn't worth the benefit